src/layout/mod.rs

```diff
@@ -57,5 +57,5 @@ impl Layout {
     }
 }
 
-pub const CORDER: u32 = 1 << 0;
+pub const CORDER: u32 = 1;
```

Comment:
IMO, it's clearer to leave it as-is or use this instead (which satisfies Clippy):

```rust
pub const CORDER: u32 = 0b01;
pub const FORDER: u32 = 0b10;
```